packages/dredd/package.json

```diff
@@ -0,0 +1,107 @@
+{
+  "name": "dredd",
+  "version": "0.0.0-semantically-released",
+  "description": "HTTP API Testing Framework",
+  "main": "build/index.js",
+  "bin": {
+    "dredd": "bin/dredd"
+  },
+  "engines": {
+    "node": ">=8"
+  },
+  "scripts": {
+    "start": "npm run build -- --watch",
+    "build": "tsc --build tsconfig.json",
+    "lint": "./scripts/commitlint.sh && eslint . bin/dredd",
+    "prettify": "prettier --write lib/**/*.js test/**/*.js",
+    "prettify:check": "prettier --check lib/**/*.js test/**/*.js",
+    "test": "mocha \"./test/**/*-test.js\"",
+    "test:debug": "mocha --debug-brk \"./test/**/*-test.js\"",
+    "e2e": "npm run e2e:apib && npm run e2e:openapi2",
+    "e2e:apib": "cucumber-js",
+    "e2e:openapi2": "cucumber-js --world-parameters=\"{\\\"apiDescriptionFormat\\\": \\\"application/swagger+yaml\\\"}\"",
+    "ci:lint": "npm run lint",
+    "ci:test": "npm test",
+    "ci:smoke": "bash ./scripts/smoke.sh",
```

Comment:
I don't think this comment is about the PR itself, but once we have dredd-transactions in the repo too, we will want to alter the smoke tests to test with the in-repo dredd-transactions.

When the time comes for this, you can take a look at how I solved similiar in API Elements where "Fury CLI" uses the local in-repo libraries as part of the smoke test: https://github.com/apiaryio/api-elements.js/blob/dbd6aafbd50fcb2fab43eae9d612561147308377/.circleci/smoke.sh